rfcs/text/0000-roadmaps+planning.md

```diff
@@ -0,0 +1,421 @@
+
+For example, an implementation task for a specific component resides in its own repo and is linked to the relevant Epic in the *dcp* repo.
+
+#### User Stories
+
+User stories are often in the form of simple templates such as:
+
+`As a <type of user>
+I want to <action>
+so that <result>`
+
+DCP issues SHOULD include a User Story, Acceptance Criteria ("Definition of Done"), and a Deliverable to be demonstrated during the Release Retrospective to Oversight.
+
+The `<type of user>` MUST be specific and reference [HCA Personas](https://docs.google.com/presentation/d/1bsu8q9CzRXv3Y8c1p8NENi3p8KlxC4scp2tJkHO3Jeo/edit#slide=id.g3cf4e46e8f_0_38) when appropriate.
+
+*As a data wrangler working on the DCP, I want to adapt the metadata schema quickly in order to respond to developments in experimental approaches and new assays.*
+
+*Acceptance Criteria*
+- *[ ] This*
+- *[ ] That*
+- *[ ] And one more thing*
+
+*Deliverable that can be demonstrated to Oversight*
+
+...
+
+### Planning and Executing a Release
+
+The DCP community members SHOULD respond to requests related to Release planning and execution in a timely manner. If there is a failure to engage and be responsive, then a gentle escalation to the DCP PM slack channel is RECOMMENDED to ensure that progress occurs and the schedule does not slip. 
+
+#### Timebox for Planning and Execution
+
+Work MAY be completed at any point during the timeboxes below but MUST be completed no later than the final day of the timebox.
+
+Release planning begins in the third Week of the third Milestone of the current Release. For example Q4 planning begins in Q3M3W3 and continues into Q4M1. To reiterate, Planning continues into Milestone 1 of a Release as needed.
+
+* Prerequisite: Product Roadmap
+* Prerequisite: Technical Objectives RFC
+* Previous Release Milestone 3 Week 3: [Clarifying the Next Release](#clarifying-the-next-release)
+* Next Release Milestone 1 Week 1: [Sketching the Release](#sketching-the-release)
+* Next Release Milestone 1 Week 2: [Creating the Engineering Plan](#creating-the-engineering-plan)
+
+#### Clarifying the Next Release
+[Clarifying the Next Release]:#clarifying-the-next-release
+
+***Previous Release Milestone 3 Week 3.*** The Roadmap Shepherd schedules a two-hour meeting with DCP PM and the Technical Leads to review any incomplete Objectives in the Release, the next Objectives in the DCP Roadmap, and the Technical Objectives RFC.
+
+Incomplete Objectives are the highest priority for the next Release unless emerging conditions have altered the priorities for the previous Release plan.
+
+The outcome from this meeting is the proposed set of Product and Technical Objectives for the next Release. One (and only one) Product Owner MUST be assigned to each proposed Objective. This Product Owner is responsible for refining and driving the Objective to on-time closure in coordination with Technical Leads and other collaborators.
```

Comment:
How does the assigned PO ensure they have resources to complete the objective in cases where the objective deals with work in "the commons" of the DCP?

Reply:
The Product Owner is responsible for ensuring that the right conversations occur with collaborators. The Technical Leads then need to estimate fit and identify capacity conflicts. And then _Based on rough costing, Technical Leads and their engineering teams SHOULD provide an initial assessment for “fit” - whether any proposed Epics need to be cut or reduced in scope for the Release._  - then later _This Epic represents the Engineering Plan which MUST be sent to the DCP PM mailing list for review and approval, noting any Objectives that were cut or reduced in scope due to engineering capacity._

Does this address your question?

Reply:
Yes, thanks.